scripts/configure-hypervisor.sh

```diff
@@ -283,13 +283,11 @@ generate_qemu_options() {
 		qemu_options+=(misc:--static)
 	fi
 
-	# Disable debug and "-uuid ..." is always passed to the qemu binary so not required.
```

Comment:
any idea about in which release this option was deprecated?

Reply:
@devimc Please refer to commit 315d318 of qemu
git describe --contains 315d318
v2.8.0-rc0~121^2~14